helix-core/src/syntax/overlay/test.rs

```diff
@@ -0,0 +1,32 @@
+use crate::syntax::{overlapping_overlay, Highlight, HighlightEvent::*};
```

Comment:
For such a small test file I think inlining the module into `helix_core::syntax::overlay`'s file is appropriate and in-line with the rest of the unit tests in the codebase.

Reply:
fixed in 1f26bca